src/dashboard/DashboardPage.vue

```diff
@@ -1,11 +1,16 @@
 <template lang="pug">
 div.container
-  div.mb-2
-    b-alert(show).mt-2.mb-0.p-2
-      div(v-if="netId === -1")
+  a(v-on:click='toTop()', v-if='dismissedErrors < errors.length', style='position:fixed;bottom:20px;right:100px;z-index:100')
+    font-awesome-icon(icon="exclamation-triangle", size='3x', style='cursor:pointer').text-warning
+  div
+    b-alert(show)
+      div(v-if='netId === -1')
         | You are not connected to an Ethereum Network. Please install this extension: #[a(href='https://metamask.io/') https://metamask.io/].
       div(v-else)
-        | You are connected to the {{ netName }}
+        | You are connected to the 
```

Comment:
Invisible trailing space, please put the two lines of text on the same line: `| You are connected to the {{ netName }}`